projects/js-packages/components/components/alert/index.tsx

```diff
@@ -43,9 +53,13 @@ const getIconByLevel = level => {
  * @param {string} props.level             - The alert level: error, warning, info, success.
  * @param {boolean} props.showIcon         - Whether to show the alert icon.
  * @param {React.Component} props.children - The alert content.
- * @returns {React.Component}                The `Alert` component.
+ * @returns {React.ReactElement}             The `Alert` component.
  */
-function Alert( { level, children, showIcon } ) {
+const Alert: React.FC< React.PropsWithChildren< AlertProps > > = ( {
+	level,
+	children,
+	showIcon,
+}: AlertProps ): React.ReactElement => {
```

Comment:
You don't need to duplicate `AlertProps` and `React.ReactElement` since you're using `React.FC`: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L517-L525

```typescript
const Alert: React.FC< React.PropsWithChildren< AlertProps > > = ( {
	level,
	children,
	showIcon,
} ) => { ... }
```

Reply:
thanks 🙇‍♂️